core/text/phrase_ender.talon-list

```diff
@@ -0,0 +1,8 @@
+list: user.phrase_ender
+-
+
+over:          ""
+space:         " "
+question:      "?"
+bang:          "!"
+slap:          "\n"
```

Comment:
This is a bit different from the standard `slap` (`edit.line_insert_down()`) behavior if you're in the middle of a line, right?

The newline will be inserted directly after the phrase, breaking the line, whereas slap normally inserts anewline after the end of the current line. If you're already at the end I think it behaves the same.

Reply:
True, but this is just about ending with a particular symbol. Adding support for edit actions in the middle of a phrase is possible, but definitely more complicated the current implementation. We could call it something else if you think the overloaded phrase is confusing. This is just what I'm personal running with.